recipes/django-jsoneditor/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "django-jsoneditor" %}
+{% set version = "0.0.12" %}
+{% set build = 0 %}
+{% set sha256 = "5f38e808cf6148fb38309551e8c76a19c6bff2403aff8456e798fd4944a97127" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/ae/b0/0164e082b29777fbc56c2373b68da93d23a29a040787e7f1c65e1562f133/django-jsoneditor-0.0.12.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: {{ build }}
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - packaging
+    - pip
+
+  run:
+    - python
+    - django
+
+test:
+  imports:
+    - jsoneditor
+    - jsoneditor.fields
+
+about:
+  home: https://github.com/nnseva/django-jsoneditor
+  summary: 'Django JSON Editor'
+  dev_url: https://github.com/nnseva/django-jsoneditor
+  license: LGPL-3.0
+  #license_file: LICENSE  # When 0.0.13 is released the LICENSE file will be available
```

Comment:
For now can you manually package the `license_file`?

https://conda-forge.org/docs/meta.html#packaging-the-license-manually